modules/triggers-supported-filter-dialects.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
+//
+// * /serverless/eventing/triggers/new-trigger-filters.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="triggers-supported-filter-dialects_{context}"]
+= Supported filter dialects
```

Comment:
This seems to render as the first header in the ToC.

I'd like to have the other doc header being "overview" or "background" (or the like)

Reply:
+1

Reply:
Added